ci-operator/step-registry/syndesisio/deploy-syndesis-qe/syndesisio-deploy-syndesis-qe-commands.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "Deploying Syndesis QE Test Runner"
+
+oc project default
+
+ADMIN_PASSWORD=$(cat "$SHARED_DIR"/kubeadmin-password)
+export ADMIN_PASSWORD
+
+URL=$(oc whoami --show-server)
+export URL
+
+cat <<EOF | oc create -f -
+apiVersion: v1
+kind: Pod
+metadata:
+  name: test-runner
+  namespace: default
+spec:
+  restartPolicy: Never
+  securityContext:
+    seccompProfile:
+      type: RuntimeDefault
+  containers:
+    - resources:
+        requests:
+          cpu: "100m"
+          memory: "200Mi"
+      name: cli
+      command:
+        - /bin/sh
+        - '-c'
+        - sleep 7200
+      securityContext:
+        capabilities:
+          drop:
+            - ALL
+        runAsUser: 1200
+        runAsGroup: 1201
+      volumeMounts:
+        - name: test-run-results
+          mountPath: /test-run-results
+      image: >-
+        $CLI
+    - resources:
+        requests:
+          cpu: "100m"
+          memory: "2Gi"
+      name: runner
+      env:
+        - name: ADMIN_USERNAME
+          value: kubeadmin
+        - name: ADMIN_PASSWORD
+          value: $ADMIN_PASSWORD
+        - name: NAMESPACE
+          value: $FUSE_ONLINE_NAMESPACE
+        - name: URL
+          value: $URL
+      securityContext:
+        capabilities:
+          add:
+            - AUDIT_WRITE
+        runAsUser: 1200
+        runAsGroup: 1201
+      volumeMounts:
+        - name: test-run-results
+          mountPath: /test-run-results
+      image: >-
+        $FUSE_ONLINE_TEST_RUNNER
+      workingDir: /home/seluser/syndesis-qe
+  volumes:
+    - name: test-run-results
+EOF
+
+oc wait --for=condition=Ready pod/test-runner --timeout=15m || true
+oc logs -c runner -n default -f test-runner || true
+oc wait --for=condition=ContainersReady=false pod/test-runner --timeout=1h30m || true
```

Comment:
why these statements are set to always true? what happens if the pod creation fails?